.github/actions/build-test-llvm-project/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and test llvm-project
+description: Build and test an llvm-project container
+
+inputs:
+  file:
+    description: Dockerfile to build the container from
+    required: true
+  platforms:
+    description: Platforms to build the container for
+    required: true
+  tags:
+    description: Tags to use for the final container
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Build llvm-project
+      uses: docker/build-push-action@v3
```

Comment:
This is an odd way to build, but I see that its just a shuffle, so :man_shrugging: